src/sage/modular/multiple_zeta_F_algebra.py

```diff
@@ -11,6 +11,9 @@
 Here we provide a basic direct implementation, endowed with the
 motivic coproduct.
 
+The similar algebra on generators `f_1`, `f_3`, `f_5`, ... and `f_2`
+is now also available.
```

Comment:
```suggestion
The similar algebra with generators a subset of
`f_1`, `f_3`, `f_5`, `\ldots` and `f_2` is now also available.
```
Perhaps this is closer to what you mean? The current addition makes it seem like it is just the same algebra as it is generated by the same set (unless I am missing something).

Reply:
no, but I wrote something vague, for sure. There is a base ring which remains a polynomial in f_2, over which one has a shuffle algebra in one more generator f_1, in addition to the generators f_3, f_5, etc. Should I reformulate ?